system/host_cpu.tf

```diff
@@ -0,0 +1,33 @@
+resource "datadog_monitor" "cpu_average" {
+  name    = "${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High CPU usage last ${var.cpu_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
```

Comment:
This is incompatible with autoscaling groups. Can you see if they have some macros?